README.md

```diff
@@ -370,13 +370,36 @@ You can alternatively mount an extra  config file into the setting's folder i.e
 ```shell
 docker run --name "postgis" -v /data/extra.conf:/settings/extra.conf -p 25432:5432 -d -t kartoza/postgis
 ```
+The `/setting` folder stores the extra configuration and is copied to the proper directory
+ on runtime. The environment variable `EXTRA_CONF_DIR` controls the location of the mounted
+ folder.
+
+Then proceed to run the following:
+
+```shell
+ docker run --name "postgis" -e EXTRA_CONF_DIR=/etc/conf_settings -v /data:/etc/conf_settings -p 25432:5432 -d -t kartoza/postgis
+```
 
 If you want to reinitialize the data directory from scratch, you need to do:
 
 1. Do backup, move data, etc. Any preparations before deleting your data directory.
 2. Set environment variables `RECREATE_DATADIR=TRUE`. Restart the service
 3. The service will delete your `DATADIR` directory and start reinitializing your data directory from scratch.
 
+### Lockfile
+
+ During container startup, some lockfile are generated which prevent reinitialisation of some
+ settings. These lockfile are by default stored in the `/settings` folder, but a user can control
+ where to store these files using the environment variable `CONF_LOCKFILE_DIR` Example
+
+ ```
+ -e /opt/conf_lockfiles \
```

Comment:
I think you are missing the env name here

Reply:
Fixed